_component_design/navigation.md

```diff
@@ -1,9 +1,10 @@
 ---
 title: Navigation
-layout: default
-category: Layout
+parent: null
+layout: component
+category: Components
 usage: Teaser Text
-preview-image: layout/preview-images/navigation.svg
+preview-image: components/preview-images/navigation.svg
```

Comment:
This is the relative link to the Cloud Files CDN where the preview images are held; we need to move the image from the `layouts` container to the `components` container if we're going to change the path.

Reply:
So.... the svg's on cloud files are not in a standardized folder structure. New images that have been created are starting to fall into more logical folders but going back and restructuring all images and then updating pages has not been done. So there are two options, do incremental changes like this, and move files as we edit a page and associated meta-data or we hold off on worrying about moving these files around and just leave them alone for now and then restructure all svgs at once. @araiford thoughts?

Reply:
@technabors my understanding is that the design docs surge effort is intended to raise every one of our articles to the same bar.  I lean towards moving or cleaning the SVGs and file structures on an article-by-article basis, as long as we have a convention for folder structures and naming on Cloud Files.

Reply:
@nimbinatus thanks for the context. I wondered about the image dir when I made the commit. It didn't seem to throw an error, but I was sure there was more to it.

@technabors @araiford who would move the graphics? If that should be the doc writers, may we have an agreed-upon procedure (with naming conventions) to follow?

Reply:
@robb-romans @technabors -- it may not matter who moves the files, but a procedure and naming conventions are definitely required.  We can define those together I think.